app/controllers/users_controller.rb

```diff
@@ -1,10 +1,10 @@
 class UsersController < ApplicationController
   def index
-    render 'index'
+    @user = User.all
   end
 
   def show
-    render 'show'
+    @post = User.recent_posts(params[:id])
```

Comment:
- [ ] **[OPTIONAL]** We can improve the `recent_posts` scope in the User model and make it just a method. Then we can call it here. So we don't need to be using `posts[0]` in the views

  ```
  def recent_posts
    posts.order(created_at: :desc).limit(5)
  end
  ```

  This way we can just call the users' posts in the view.

  ```
  @user = User.find(params[:id)
  ```

  Then in the show view we can present the recent posts:

  ```
  @user.recent_posts
  ```